### DISCUSSION TITLE
Simulating different stages of aging in one pack

### DISCUSSION THREAD

**Discussion by bilgermoritz** [pybamm-team/liionpack#277] [Q&A] [answered]
Hi there, 
Is there a possibility to parametrize different aging stages (e.g. SEI parameters) in one pack? I am writing my masters thesis on second life batteries and therefor heterogeneity of aging stages is a very interesting topic to me.  
Maybe you can give me an estimation if that is possible or not and where you would implement such a functionality in the code. Some critical thoughts on how this would affect solving accuracy would also be helpful. 
Thanks in advance!

**Comment by TomTranter** [comment, marked as answer]
Hi @bilgermoritz,
Do you mean age some battery models differently and then make a pack out of them? Should be possible if you play with the ageing models for a single cell then capture the states that have changed and use them as inputs to the pack model. For example doing a loss of active material could be simply modelled by setting the active material volume fraction as an input in the pack model and passing an array of different values. You would have to understand what has changed in a single cell depending on what degradation mechanisms you want to model but the main things to control manually are active material, total cyclable lithium, SEI thickness, porosity

**Reply by bilgermoritz** [reply]
That is exactly what i meant! Thank you for your response!
